Makefile

```diff
@@ -14,7 +14,9 @@ help:
 	echo $(BUILD_DATE)
 
 godeps:
+ifneq ($(FAST),1)
```

Comment:
Used only in development like: `FAST=1 make`. Only works if you already have `godep` installed.